packages/buidler-core/src/internal/cli/cli.ts

```diff
@@ -176,7 +176,7 @@ async function main() {
 }
 
 main()
-  .then(() => process.exit(0))
+  .then(() => process.exit(process.exitCode))
```

Comment:
Passing the exitCode here is not necessary

Reply:
As discussed IRL we'll keep it to be more explicit.